pom.xml

```diff
@@ -657,7 +657,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.build.outputEncoding>UTF-8</project.build.outputEncoding>
 		<lombokVersion>1.18.10</lombokVersion>
-        <openmrsPlatformVersion>2.3.0</openmrsPlatformVersion>
+        <openmrsPlatformVersion>2.0.2</openmrsPlatformVersion>
```

Comment:
This should be 2.0.5. Also, please ensure your IDE is configured to indent XML files with tabs.

Reply:
Ok will make this change